entrypoint.sh

```diff
@@ -1,8 +1,25 @@
 #!/bin/sh
+version() {
+  if [ -n "$1" ]; then
+    echo "-v $1"
+  fi
+}
+
+install_extensions() {
+  if [ -n "$1" ]; then
+    echo "$1" | sed "s/[^ ][^ ]*/'&'/g" | xargs gem install
```

Comment:
Can we install extensions via the command: `gem install -N $INPUT_RUBOCOP_VERSION`?

Reply:
No we cannot specify version of gem with -N flaf

Reply:
Sorry, I mean the command to install extensions: `gem install -N $INPUT_RUBOCOP_EXTENSIONS `

Reply:
I get this result
```sh
INPUT_RUBOCOP_EXTENSIONS='rubocop-performance:1.5.1 rubocop-minitest'
gem install -N $INPUT_RUBOCOP_EXTENSIONS
ERROR:  Could not find a valid gem 'rubocop-performance:1.5.1 rubocop-minitest' (>= 0) in any repository
```
but I guessed that everything is installed with a simple parameter passing through xargs
```sh
echo $INPUT_RUBOCOP_EXTENSIONS | xargs gem install -N                
Successfully installed rubocop-performance-1.5.1
Fetching rubocop-minitest-0.9.0.gem
Successfully installed rubocop-minitest-0.9.0
2 gems installed
```